maxim/logger/crewai/client.py

```
@@ -9,7 +9,7 @@
 
 from crewai import LLM, Agent, Crew, Flow, Task
 from crewai.agents.agent_builder.base_agent import BaseAgent
-from crewai.tools.base_tool import BaseTool
+from crewai.tools.agent_tools import BaseTool
```

Pylint (3.3.7)

[error] Unable to import 'crewai.tools.agent_tools'

(E0401)